test/context/cookies.js

```diff
@@ -1,35 +1,26 @@
 
 'use strict';
 
-const request = require('supertest');
+const request = require('../helpers/request');
 const Koa = require('../..');
 
 describe('ctx.cookies.set()', () => {
-  it('should set an unsigned cookie', done => {
+  it('should set an unsigned cookie', async () => {
     const app = new Koa();
 
     app.use((ctx, next) => {
       ctx.cookies.set('name', 'jon');
       ctx.status = 204;
     });
 
-    const server = app.listen();
-
-    request(server)
-      .get('/')
-      .expect(204)
-      .end((err, res) => {
-        if (err) return done(err);
-
-        res.headers['set-cookie'].some(cookie => /^name=/.test(cookie)).should.be.ok;
-
-        done();
-      });
+    const res = await request(app, '/');
```

Comment:
Looking into it, it does have a new maintainer, so my point there isn't valid. However, I still prefer the syntax of node-fetch. May I ask why you prefer supertest?